_pages/calls/demos.md

```diff
@@ -0,0 +1,112 @@
+---
+title: EACL 2023 – Call for System Demonstrations
+hide_title: true
+layout: single
+permalink: /calls/demos/
+sidebar:
+  nav: calls
+toc: true
+toc_only: false
+toc_sticky: true
+toc_icon: "cog"
+---
+
+<h1>EACL 2023 – Call for System Demonstrations</h1>
+Submission link: [softconf.com/eacl2023/demos](softconf.com/eacl2023/demos)<br />
+
+The system demonstration track at EACL 2023 is a venue for papers describing system demonstrations, ranging from early prototypes to mature production-ready systems. Publicly available open-source or open-access systems are of special interest.
+
+All accepted demos are published in a companion volume of the conference proceedings. We expect at least one of the authors to present a live demo during a demo session at EACL 2023, with an accompanying poster, either virtually or in-person.
+
+Submissions will undergo a single-blind reviewing process. Therefore, papers may include author and affiliation information, and freely make references to previously published material, and URLs.
+
+This year we are incorporating ethical considerations in the review process. Authors will be allowed extra unlimited space after the main content for a broader impact statement or other discussions of ethics. Please review the ethics policy before submitting.
+
+
+
+## Important Dates
```

Comment:
Are these dates only reflected on the call for demos? We also have a list of important dates on the home page, but I'm not sure if they should include the demo dates as well.

Reply:
Good catch! fixed.